frontend/src/common/theme.ts

```diff
@@ -270,6 +270,9 @@ export const gray500 = grey500;
 export const grey600 = (props: CommonThemeProps) => getColors(props)?.gray[600];
 export const gray600 = grey600;
 
+export const greyWhite = () => "#fffff";
```

Comment:
Bug! `#fffff` is an invalid css hex color

Reply:
😱